### ISSUE TITLE
Starting point

### ISSUE THREAD

**Issue by msarahan** [conda/build_infrastructure#1]
@ocefpaf @pelson @mingwandroid - I'm proposing this repo as a starting place for any potential collaboration on this build system.  Let's discuss any ideas here.

I'd like to have a more detailed meeting with @ocefpaf and @pelson to discuss the details of the systems that you already have, or are planning.  Please suggest a day/time - My mornings are always free before 11 AM CT.

**Comment by ocefpaf** [comment]
> My mornings are always free before 11 AM CT.

I am two hours ahead  CT, so anytime in your morning is fine by me.

@kwilcox do you want to participate? Or maybe ask someone from Axiom to?

**Comment by kwilcox** [comment]
I'd love to. Eastern 9-5 work for me.

**Comment by msarahan** [comment]
Excellent!  I have proposed a meeting on Monday at 9:30 eastern/8:30 central: https://plus.google.com/hangouts/_/continuum.io/build?hceid=bXNhcmFoYW5AY29udGludXVtLmlv.utoh79q675sqvg7onv0i46ud0g&authuser=1

If this does not work, I'm happy to move it.

**Comment by jakirkham** [comment]
Would be curious to hear what came out of this.

**Comment by msarahan** [comment]
@kwilcox suggested Gitlab CI.  I have played with Buildbot, DroneCI, Gitlab CI, and minimally with Anaconda Build.  Gitlab CI seems the closest to our goals.  I don't think we can use the CI component separately from the hosting component, but I think it will be OK - basically, we will mirror github repos, and trigger mirroring operations with particular repo hooks.

I have been working on the Vagrant Windows VM.  I have Mac 10.7 and 10.9 OVAs that I'm planning to also use with vagrant.  I don't know yet what the license restrictions might be on redistribution of these images.  We have to run mac VM's on Mac hardware, but I don't know anything about who (if anyone) can host VM images.

**Comment by jakirkham** [comment]
This sounds very promising, @msarahan! I'd love to hear more about this in our meeting and am sure others would too.

It would also give us a good opportunity to ask who can lend infrastructure and what kinds. Maybe the answer doesn't come right a way as some investigation may need to follow. That all being said, we still have to iron out how we would do this.

**Comment by jakirkham** [comment]
cc @ericdill

**Comment by jakirkham** [comment]
Are you referring to Vagrant Windows VM from PR ( https://github.com/ContinuumIO/vagrant-images/pull/1 )? What still needs to be done with it? Is it ready to go?

**Comment by msarahan** [comment]
Yes.  It's mostly ready.  It's a matter of making sure that all the compilers are installed and working.  At the moment, I think they are all working, but miniconda will not install - it seems to hang.  I need to push up what I have.  I will try to do so soon.

**Comment by jakirkham** [comment]
Thanks for continuing to working on this, @msarahan. 😄

**Comment by jakirkham** [comment]
Also would be very curious to see the OS X `Vagrantfile`s even if we might not have the OVAs yet due to ambiguity with the license.

**Comment by msarahan** [comment]
I have not begun working on OSX.  I'll ping you when I have something to show.

**Comment by jakirkham** [comment]
Have you seen [this](https://github.com/boxcutter/macos), @msarahan ?

**Comment by msarahan** [comment]
I had not. Thanks for pointing it out. We have an IT person working on
this. I will show him.

**Comment by jakirkham** [comment]
So I just used their repo to create a 10.9 image using my `InstallESD.dmg`. It's probably the most painless process I have gone through setting up a VM to date. I still have no clue how one goes about distributing an image like this legally ( IANAL ).
